projects.jsonl

```diff
@@ -0,0 +1,48 @@
+{"shortname": "experimental/coref", "title": "Training a spaCy Coref Model", "description": "This project trains a coreference model for spaCy using OntoNotes.\n\nBefore using this project:\n\n1. install spaCy with GPU support - see the [install widget](https://spacy.io/usage)\n2. run `pip install -r requirements.txt`\n3. modify `project.yml` to set your GPU ID and OntoNotes path (see [Data Preparation](#data-preparation) for details)\n\nAfter that you can just run `spacy project run all`.\n\nNote that during training you will probably see a warning like `Token indices\nsequence length is longer than ...`. This is a rare condition that\n`spacy-transformers` handles internally, and it's safe to ignore if it\nhappens occasionally. For more details see [this\nthread](https://github.com/explosion/spaCy/discussions/9277#discussioncomment-1374226).\n\n## Data Preparation\n\nTo use this project you need a copy of [OntoNotes](https://catalog.ldc.upenn.edu/LDC2013T19).\n\nIf you have OntoNotes and have not worked with the CoNLL 2012 coreference annotations before, set `vars.ontonotes` in `project.yml` to the local path to OntoNotes. The top level directory should contain directories named `arabic`, `chinese`, `english`, and `ontology`. Then run the following command to prepare the coreference data:\n\n```\nspacy project run prep-conll-data\n```\n\nAfter that you can execute `spacy project run all`.\n\nIf you already have CoNLL 2012 data prepared and concatenated into one file per split, you can specify the paths to the training, dev, and test files directly in `project.yml`, see the `vars` section. After doing so you can run `spacy project run all`.\n\n## Using the Trained Pipeline\n\nAfter training the pipeline (or downloading a pretrained version), you can load and use it like this:\n\n```\nimport spacy\nnlp = spacy.load(\"training/coref\")\n\ndoc = nlp(\"John Smith called from New York, he says it's raining in the city.\")\n# check the word clusters\nprint(\"=== word clusters ===\")\nword_clusters = [val for key, val in doc.spans.items() if key.startswith(\"coref_head\")]\nfor cluster in word_clusters:\n    print(cluster)\n# check the expanded clusters\nprint(\"=== full clusters ===\")\nfull_clusters = [val for key, val in doc.spans.items() if key.startswith(\"coref_cluster\")]\nfor cluster in full_clusters:\n    print(cluster)\n```\n\nThe prefixes used here are a user setting, so you can customize them for your\nown pipelines.\n"}
```

Comment:
Did you lint/validate the `.jsonl` file to make sure it's valid? It looks alright to me at first glance, I'm just confused as to why the GH syntax highlighting doesn't work in some lines.

Reply:
Confirmed it's OK with jq. I GH just doesn't highlight very long lines, for example this is a little over 1000 "a"s:

```jsonl
{"ok": "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"}
```

Reply:
Thanks for checking!